shared/middleware/net_sim_api.rb

```diff
@@ -6,15 +6,21 @@
 class NetSimApi < Sinatra::Base
 
   helpers do
-    [
-        'core.rb',
-        'storage_id.rb',
-        'table.rb',
-    ].each do |file|
+    %w{
+      core.rb
+      storage_id.rb
+      table.rb
+      null_pub_sub_api.rb
+      pusher_api.rb
+    }.each do |file|
       load(CDO.dir('shared', 'middleware', 'helpers', file))
     end
+
   end
 
+  # For test, make it possible to override the usual configured API choice
+  @@overridden_pub_sub_api = nil
```

Comment:
Please add a "_for_test" suffix to this class variable and to the override_pub_sub_api method so that no one ever accidentally calls this in production.